.ci/pitest.sh

```diff
@@ -53,22 +53,21 @@ pitest-annotation|pitest-design \
 |pitest-tree-walker \
 |pitest-utils \
 |pitest-java-ast-visitor)
-  mvn --no-transfer-progress -e -P"$1" clean test-compile org.pitest:pitest-maven:mutationCoverage;
+  mvn -e --no-transfer-progress -P"$1" clean test-compile org.pitest:pitest-maven:mutationCoverage;
```

Comment:
Had to reverse the order here.

Reply:
see https://dev.azure.com/romanivanovjr/a35b6275-caac-4633-b371-79a5ab0f70c1/_apis/build/builds/9538/logs/89